apps/weshipit/components/layout.tsx

```diff
@@ -1,4 +1,5 @@
 import { Header } from '@weshipit/ui';
+import { Inter } from '@next/font/google';
```

Comment:
we should not have this here

Reply:
<img width="830" alt="Capture d’écran 2022-11-15 à 13 31 41" src="https://user-images.githubusercontent.com/81434852/201920475-01d42a05-f8b9-4072-8985-edded1b6e617.png">